pyroclast/prototype/prototype.py

```diff
@@ -96,17 +113,101 @@ def run_minibatch(epoch, batch, is_train=True):
                                   l1_coeff *
                                   tf.reduce_mean(loss_term_dict['l1']),
                                   step=global_step)
+            if 'separation' in loss_term_dict:
+                tf.summary.scalar(prefix + "loss/mean separation",
+                                  separation_coeff *
+                                  tf.reduce_mean(loss_term_dict['separation']),
+                                  step=global_step)
             tf.summary.scalar(prefix + "loss/total loss",
                               loss,
                               step=global_step)
 
+    ### PHASE 1
     # run training loop
-    for epoch in range(epochs):
+    if debug: print("PHASE 1 - TRAINING CONV STACK AND PROTOTYPES")
+    for epoch in range(epochs_phase_1):
```

Comment:
Shouldn't this block of code only run if phase is equal to one?

Reply:
It's the part of the code that actually runs the first phase. Ditto for the following 2 sections as well.

Reply:
That's what I'm saying. The first phase should be run if the phase argument is set to 1, right? The argument doesn't seem to actually do what it's intended to do.